en/config/actuators.md

```diff
@@ -1,139 +1,483 @@
+1. Return the slider to the minimal position.
+1. Repeat for all motors
+
+Actuator assignment can be checked in a similar way.
+
+
+### Motor Configuration
+
+The motor configuaration sets output values such that motors:
+- don't spin when disarmed (at the `disarmed` PWM output value)
+- barely spin at the `minimum` PWM output value
+- give **positive thrust** in the expected direction
+
+For each motor:
+
+1. Pull the motor slider down so that it snaps to the bottom.
+   In this position the motor is set to the outputs `Disarmed` value.
+   - Verify that the motor doesn't spin in this position.
+   - If the motor spins, reduce the corresponding PWM `Disarmed` value in the [Actuator Outputs](#actuator-outputs) section to below the level at which it still spins.
+2. Slowly move the slider up until it snaps to the _minimum_ position.
+   In this position the motor is set to the outputs `Minimum` value.  
+   - Verify that the motor is spinning very slowly in this position.
+   - If the motor is not spinning, or spinning too fast you will need to adjust the corresponding PWM `Disarmed` value in the [Actuator Outputs](#actuator-outputs) such that the motors barely spin.
+   
+     ![PWM Minimum Output](../../assets/config/actuators/pwm_minimum_output.png)
+     :::note
+     For DShot output, this is not required <!-- any, or just the minimum check? -->
+     :::
+3. Increase the slider value to a level where you can verify that the motor is spinning in the correct direction and that it would give a positive thrust in the expected direction.
+   - The expected thrust direction can vary by vehicle type, for example in multicopters the thrust should always point upwards, while in a fixed wing vehicle the thrust will push the vehicle forwards.
+   - For VTOL, thrust should point upwards when the Tilt Servo is at 0 degrees as defined the [Tilt Servo Convention](#motor-tilt-servos-conceptual).
+     Testing of the [Tilt Servo](#tilt-servo-testing) is covered below as well.
+   - If thrust is in the wrong direction, you may need to [reverse the motors](#reversing-motors).
+   
+
+### Control Surface Setup
+
+The control surface configuaration sets output values such that the surfaces:
+
+1. have the correct idle position when disarmed.
+1. move in the directions as defined in the [Control Surface Convention](#control-surface-direction-conceptual).
+
+![Control Surface Disarmed 1500 Setting](../../assets/config/actuators/control_surface_disarmed_1500.png)
+
+For each control surface:
+1. Set the `Disarmed` value for Control Surface outputs to `1500` (note, this assumes we're using a PWM actuator where 1500 is the middle of the range)
+   - Unlike motors, when `disarmed` we usually control surfaces to be in the middle position of their range.
+   - Adjust the value until the control is positioned correctly.
+2. Set the Output mode to `PWM 50Hz`, instead of default PWM 400 Hz when using a PWM output for Control Surfaces
```

Comment:
What? Now you're saying I change my mode, and now I use PWM50Hz rather that PWM400Hz? This makes no sense.

What I "think" you mean is that you generally don't need as high speed control for actuators, so you're recommending a slow protocol to get more reliability. Is that right?

So I think you need to start by saying all of that BEFORE you have any of these steps. And then describe the rest of the setting relative to just one PWM setting.

Reply:
Gotcha, and yes it's to prevent the Servo from making weird noises when it's commanded with PWM at 400 Hz. And because the default configuration is 400 Hz, I added the command there :thinking: 

Since it applies to the whole commands, I agree it's better to put it in the beginning.